test/utils/common.js

```diff
@@ -165,6 +166,10 @@ export async function createSheepReviewClaim(
     await clickContinueButton();
   }
 
+  if (isUnnamedHerdClaimPresent) {
+    await clickOnElementAndContinue(getSelectHerdSelector("a different "));
```

Comment:
Can we change "a different " to more closely align to what the text will say? I think it will always say "No, it's a different flock" wont it?